src/sagemaker_xgboost_container/constants/sm_env_constants.py

```diff
@@ -32,3 +32,7 @@
 SAGEMAKER_INFERENCE_OUTPUT = 'SAGEMAKER_INFERENCE_OUTPUT'
 SAGEMAKER_DEFAULT_INVOCATIONS_ACCEPT = 'SAGEMAKER_DEFAULT_INVOCATIONS_ACCEPT'
 SAGEMAKER_BATCH = 'SAGEMAKER_BATCH'
+
+# Multiprocessing related constants
+# Turn off OpenMP multi-threading, Let Python processes remain single-threaded
```

Comment:
This comment should be moved to line 48 in `serving.py` (and anywhere else we actually set the environment variable) instead of here.

`Single-thread processes by default. Multithreading can introduce significant performance overhead due to task switching.`

Reply:
Agreed, we want to make sure this is only changed in serve, not training